src/java.base/share/classes/sun/security/ssl/StatusResponseManager.java

```diff
@@ -257,7 +257,19 @@ Map<X509Certificate, byte[]> get(CertStatusRequestType type,
                     }
 
                     if (!task.isCancelled()) {
-                        StatusInfo info = task.get();
+                        StatusInfo info = null;
+                        try {
+                            info = task.get();
+                        } catch (ExecutionException exc) {
+                            // Check for an underlying cause available and log
+                            // that, otherwise just log the ExecutionException
+                            Throwable cause = Optional.ofNullable(
+                                    exc.getCause()).orElse(exc);
+                            if (SSLLogger.isOn && SSLLogger.isOn("respmgr")) {
+                                SSLLogger.fine("Exception during OCSP fetch: " +
+                                        cause);
+                            }
```

Comment:
Yes, I think a continue here would be a good thing to do.  It doesn't strictly need it, because if you fall into the catch block info is null and so it will eventually fall through to the next iteration.  But it does fall into the else block of the next if/else and that log message is unnecessary since it was logged earlier.  I'll add the continue.  Thanks for catching that!